eventmesh-common/src/main/java/org/apache/eventmesh/common/config/connector/mq/kafka/SourceConnectorConfig.java

```diff
@@ -32,5 +32,4 @@ public class SourceConnectorConfig {
     private String enableAutoCommit = "false";
     private String sessionTimeoutMS = "10000";
     private String maxPollRecords = "1000";
-    private int pollTimeOut = 100;
 }
```

Comment:
Can we allocate an appropriate timeout value for a source where we anticipate the RT might be significantly slower, and override the global default value?

If this feature isn't supported yet, you can implement it in a future PR. This is very important for users who use multiple sources with the same configuration files.